docs/0_internal/2_deploy_contracts.md

```diff
@@ -0,0 +1,234 @@
+    "churner": "<churner_address>",
+    "ejector": "<ejector_address>",
+    "deployer": "<deployer_address>",
+    "initalPausedStatus": 0
+  },
+  "minimumStakes": [],
+  "strategyWeights": [],
+  "operatorSetParams": [],
+  "uri": ""
+}
+```
+
+You can find an example config file in [aligned.holesky.config.json](../../contracts/script/deploy/config/holesky/aligned.holesky.config.json).
+
+> [!Note]
+> 
+> You can find the list of Mainnet strategies for the `strategyWeights` field [here](https://github.com/Layr-Labs/eigenlayer-contracts?tab=readme-ov-file#current-mainnet-deployment)
+> 
+> You can find the list of Holesky strategies for the `strategyWeights` field [here](https://github.com/Layr-Labs/eigenlayer-contracts?tab=readme-ov-file#current-testnet-deployment)
+
+#### Multisig configuration
+
+If you are using a Multisig for the contracts management (like upgrades or pauses), you need to set the Multisig address in the `permissions` sections.
+
+For example, if you are using a Multisig for the `upgrader` permission, you need to set the Multisig address in the `upgrader` field.
+
+### Deploy the contracts
+
+Once you have configured the `.env` and `DEPLOY_CONFIG_PATH` files, you can run the following command:
+
+```bash
+make deploy_aligned_contracts
+```
+
+Once the contracts are deployed, you will see the following output at `OUTPUT_PATH` file:
+
+```json
+{
+  "addresses": {
+    "alignedLayerProxyAdmin": "<aligned_layer_proxy_admin_address>",
+    "alignedLayerServiceManager": "<aligned_layer_service_manager_address>",
+    "alignedLayerServiceManagerImplementation": "<aligned_layer_service_manager_implementation_address>",
+    "blsApkRegistry": "<bls_apk_registry_address>",
+    "blsApkRegistryImplementation": "<bls_apk_registry_implementation_address>",
+    "indexRegistry": "<index_registry_address>",
+    "indexRegistryImplementation": "<index_registry_implementation_address>",
+    "operatorStateRetriever": "<operator_state_retriever_address>",
+    "registryCoordinator": "<registry_coordinator_address>",
+    "registryCoordinatorImplementation": "<registry_coordinator_implementation_address>",
+    "serviceManagerRouter": "<service_manager_router_address>",
+    "stakeRegistry": "<stake_registry_address>",
+    "stakeRegistryImplementation": "<stake_registry_implementation_address>",
+    "batcherPaymentService": "<batcher_payment_service_address>",
+    "batcherPaymentServiceImplementation": "<batcher_payment_service_implementation_address>"
+  },
+  "chainInfo": {
+    "chainId": 17000,
+    "deploymentBlock": 1628199
+  },
+  "permissions": {
+    "alignedLayerAggregator": "<aligned_layer_aggregator_address>",
+    "alignedLayerChurner": "<aligned_layer_churner_address>",
+    "alignedLayerEjector": "<aligned_layer_ejector_address>",
+    "alignedLayerOwner": "<aligned_layer_owner_address>",
+    "alignedLayerUpgrader": "<aligned_layer_upgrader_address>",
+    "pauserRegistry": "<pauser_registry_address>"
```

Comment:
here it is AlignedLayerPauser, and the PauserRegistry goes on top bc it is a contract